src/components/CartCounter/CartCounter.tsx

```diff
@@ -1,34 +1,33 @@
-import { useState } from "react";
-
 import Grid from "@mui/material/Unstable_Grid2";
 import { IconButton } from "@mui/material";
 import { Add, Remove } from "@mui/icons-material";
 
 import styles from "./CartCounter.module.scss";
 
-const CartCounter = () => {
-  const [amount, setAmount] = useState(1);
-  const plusAmount = () => {
-    setAmount(amount + 1);
-  };
-
-  const minusAmount = () => {
-    if (amount > 1) setAmount(amount - 1);
-  };
+interface CartCounterInterface {
```

Comment:
No need in `interface` word :)
```suggestion
interface CartCounterProps {
```